atlasdb-config/src/main/java/com/palantir/atlasdb/config/TimeLockRequestBatcherProviders.java

```diff
@@ -31,4 +32,6 @@ public interface TimeLockRequestBatcherProviders {
     TimeLockRequestBatcherProvider<MultiClientCommitTimestampGetter> commitTimestamps();
```

Comment:
This is a break. I will own fixing it in the only service using this (internal skiing product).